overlord/snapstate/snapstate_try_test.go

```diff
@@ -125,41 +124,3 @@ func (s *snapmgrTestSuite) TestTryUndoRemovesTryFlagLeavesClassic(c *C) {
 	defer restore()
 	s.testTrySetsTryMode(snapstate.Flags{Classic: true}, c, "confinement: classic\n")
 }
-
-func (s *snapmgrTestSuite) testTryUndoRemovesTryFlag(flags snapstate.Flags, c *C) {
```

Comment:
Hmm wondering if we should add a test for this, maybe @pedronis or @stolowski  know better.

Reply:
I thought the same, but then I noticed that a few lines above (line 110) there is already a `TestTryUndoRemovesTryFlag` method.